eng/scripts/Language-Settings.ps1

```diff
@@ -284,8 +284,8 @@ function Get-dotnet-DocsMsMetadataForPackage($PackageInfo) {
 
   New-Object PSObject -Property @{
     DocsMsReadMeName = $readmeName
-    LatestReadMeLocation = 'api/overview/azure'
-    PreviewReadMeLocation = 'api/overview/azure'
+    LatestReadMeLocation = 'api/overview/azure/latest'
+    PreviewReadMeLocation = 'api/overview/azure/preview'
     Suffix = $suffix
```

Comment:
`Suffix` should be an empty string here. If not, releasing a preview library would result in package-level overview pages being named with `-pre.md`.

Reply:
I think this can be directly tested by running a publish of the `template` pipeline.